tests/AutoReview/CommandTest.php

```diff
@@ -37,7 +37,7 @@ public function testCommandHasNameConst(Command $command): void
         static::assertNotNull($command->getDefaultName());
     }
 
-    public function provideCommandHasNameConstCases()
+    public function provideCommandHasNameConstCases(): array
```

Comment:
Can we go with `iterable` for all data providers? Because we don't really care if it's `array` or `Generator` - all we want is iterable value.

Reply:
I like to be a precise as possible, it will also help to transform all this big array return methods into real generators using `yield`